tests/v2/codecoverage/scripts/build_docker_images.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+ARCH=${ARCH:-"amd64"}
+REPO=igomez06
```

Comment:
I don't have access to the repo, and this was temporary until I could, if you could possibly help with that than we can move our custom image there rather than my own docker repo.

Reply:
asking as I'm unsure...did this ever get traction? I agree that we should move this out of personal repos if we can help it.

Reply:
I'm trying to resolve this now. In the long run we do want to move this to our rancher-test docker hub.